Opensegaapi/src/opensegaapi.cpp

```diff
@@ -749,13 +738,17 @@ extern "C" {
 		OPEN_segaapiBuffer_t* buffer = (OPEN_segaapiBuffer_t*)hHandle;
 
 		updateRouting(buffer);
+		updateBufferNew(buffer, -1, -1);
 
-		updateBufferData(buffer, -1, -1);
-		updateBufferLoop(buffer);
-		updateSynthOnPlay(buffer, -1, -1);
 		buffer->playing = true;
 		buffer->paused = false;
-		CHECK_HR(buffer->xaVoice->Start());
+
+		// Uncomment to mute music
```

Comment:
Does this only mute music? It might be possible to turn this into a config option in the future

Reply:
not sure if it works for every lindbergh game, atleast it works for initial d